client/app/domain/translations/Messages_fr_FR.xml

```diff
@@ -519,7 +519,8 @@
    <translation id="domain_configuration_dns_entry_add_step3_alert" qtlid="195048">La modification sera immédiate dans la zone DNS, mais veuillez prendre en compte le temps de propagation (maximum 24h).</translation>
 
    <translation id="domain_configuration_dns_entry_add_step3_check_warning1" qtlid="215065">Attention, un ou plusieurs conflits peuvent survenir :</translation>
-   <translation id="domain_configuration_dns_entry_add_step3_check_warning2" qtlid="519831">Le sous-domaine est déjà utilisé par un autre enregistrement. Un enregistrement CNAME n'est pas autorisé à coexister avec d'autres champs sur le même sous-domaine.</translation>
+   <translation id="domain_configuration_dns_entry_add_step3_check_warning2">Le sous-domaine est déjà utilisé par un autre enregistrement. Un enregistrement CNAME n'est pas autorisé à coexister avec d'autres champs sur le même sous-domaine. Supprimez les enregistrements suivants pour le même sous-domaine, afin de pouvoir ajouter cet enregistrement:</translation>
+   <translation id="domain_configuration_dns_entry_add_step3_conflicting_record">{0} {1} {2}</translation>
```

Comment:
It doesn't make sense to translate this kind of sentence.

Prefer using ngBind on your view.

Reply:
This is done :)